server/core/templates/admin/admin_group_view_partial.html

```diff
@@ -0,0 +1,116 @@
+        <button hx-target="#main" hx-get="/ui/admin/group/(( group.uuid ))/edit" type="button" class="btn btn-outline-secondary">Edit Group</button>
+    (% endif %)
+    <button hx-target="#main" hx-post="/ui/admin/group/(( group.uuid ))/delete" hx-confirm="Are you sure you want to delete group (( group.name )) ?" type="button" class="btn btn-danger">Delete Group</button>
+(% else %)
+    <a hx-boost="false" href="/ui/admin/groups/unlock">
+        <button type="button" class="btn btn-secondary">Unlock Edit 🔒</button>
+    </a>
+(% endif %)
+
+(% include "admin_group_details_partial.html" %)
+
+<hr>
+<label class="mt-3 fw-bold">Emails</label>
+<form hx-validate="true" hx-ext="bs-validation">
+(% if group.mails.len() == 0 %)
+    <p>none</p>
+(% else %)
+    <ol class="list-group col-12 col-md-8 col-lg-6">
+        (% for mail in group.mails %)
+        <li id="groupMail(( loop.index ))" class="list-group-item d-flex flex-row justify-content-between">
+            <div class="d-flex align-items-center">(( mail ))</div>
+            <div class="buttons float-end">
+                (% if can_edit %)
+                <button hx-target="#groupMail(( loop.index ))"
+                        hx-swap="delete"
+                        hx-delete="/ui/admin/group/(( group.uuid ))/mail/(( mail ))"
+                        hx-confirm="Are you sure you want to remove (( mail )) from (( group.name ))?"
+                        type="button" class="btn btn-danger">Remove</button>
+                (% endif %)
+            </div>
+        </li>
+        (% endfor %)
+        (% if can_edit %)
+            <li class="list-group-item"  id="addEmailField">
+                <div class="input-group d-flex flex-row justify-content-between">
+                    <input type="email" name="mail" class="form-control" id="groupAddEmailInput" placeholder="Group's Email address" aria-label="Group's Email address" aria-describedby="groupAddEmailBtn">
+                    <button id="groupAddEmailBtn" class="btn btn-success text-nowrap"
+                            hx-post="/ui/admin/api/group/(( group.uuid ))/mail"
+                            hx-include="#groupAddEmailInput"
+                            hx-target="#addEmailField"
+                            hx-swap="beforebegin"
+                            type="button">
+                        Add Email
+                    </button>
+                </div>
+                <div class="invalid-feedback">
+                    Please choose a valid email.
+                </div>
+            </li>
+        (% endif %)
+    </ol>
+(% endif %)
+</form>
+
+<label class="mt-3 fw-bold">Account Policy (ACP)</label>
+<div>
+(% if group.acp.enabled %)
+    Nothing here yet 🙈🙈
+(% else %)
+    Currently disabled.
+    (% if can_edit %)
+        <button hx-post="/ui/api/group/(( group.uuid ))/enable_acp" hx-confirm="Currently no known way to disable again, ok to continue." class="btn btn-success d-block">Enable</button>
```

Comment:
The message here is confusing, perhaps:

```suggestion
        <button hx-post="/ui/api/group/(( group.uuid ))/enable_acp" hx-confirm="Kanidm does not currently support removing these policies, are you sure you want to continue?" class="btn btn-success d-block">Enable</button>
```